src/zui/ZUICreatePerson/PersonFieldInput.tsx

```diff
@@ -60,5 +70,18 @@ const PersonFieldInput: FC<PersonFieldInputProps> = ({
       value={value}
     />
   );
+
+  return editMode ? (
+    textField
+  ) : (
+    <Box alignItems="flex-start" display="flex" flex={1}>
+      {textField}
+      {hasChanges && (
+        <IconButton onClick={onReset} sx={{ paddingTop: 2 }}>
+          <UndoIcon />
+        </IconButton>
+      )}
+    </Box>
+  );
```

Comment:
I think the logic here is backwards and that's why the "Undo" icon is not shown in the interface!